server/remote/gitlab3/gitlab_test.go

```diff
@@ -49,15 +49,15 @@ func Test_Gitlab(t *testing.T) {
 		g.Describe("AllProjects", func() {
 			g.It("Should return only non-archived projects is hidden", func() {
 				gitlab.HideArchives = true
-				_projects, err := gitlab.Repos(&user)
+				_projects, err := gitlab.Repos(nil, &user)
```

Comment:
Why don't you use `ctx.Background()` in this file?

Reply:
because gitlab3 get deleted soon